### ISSUE TITLE
Should build_remote_directory.py accept multiple collection systems?

### ISSUE THREAD

**Issue by rgov** [OceanDataTools/openvdm#51]
The copy of this script recovered from R/V *Odyssey* adjusts the `--collection_system` flag to accept multiple parameters. I do not see this change upstream.

```diff
-    parser.add_argument('-c', '--collection_system', dest='collection_system', metavar='collection_system',
-                        help='create the remote directory/file just for the specified collection system')
+    parser.add_argument('-c', '--collection_system', nargs='+', dest='coll_systems', metavar='collection_system',
+                        help='create the remote directory/file for the specified collection system')
```

Note that `action='append'` might be better than `nargs='+'`.

**Comment by webbpinner** [comment]
That was likely something added just for Odyssey but there's no harm in adding it to the default script.  I've updated the script in the dev branch.  Will merge dev to master in the near future along with some other updates.

**Comment by rgov** [comment]
Fixed in f6d7705. Recommend avoiding conflating multiple changes in a single commit to avoid future issues with `git log` / `git blame` / `git bisect` etc.
